.github/workflows/tests_with_coverage.yml

```diff
@@ -55,4 +64,10 @@ jobs:
     steps:
       - name: Tests
         run: |
-          echo "Slow tests ran successfully"
+          echo "Slow tests approved"
+  Test-Db-Versions-All-Tests:
+    needs: Slow-Tests
+    uses: ./.github/workflows/test_db_versions_all_tests.yml
+  Test-Python-Versions:
+    needs: Slow-Tests
+    uses: ./.github/workflows/test_python_version.yml
```

Comment:
I find the name `test_python_version.yml` of the script not very expressive. 

The job inside says 
```
name: Run Tests with Python ${{ matrix.python_version }} and Exasol ${{ matrix.exasol_version }}
```

Still, I cannot tell, if these tests overlap with other groups of tests (e.g. include unit tests) or in which way they are different. Additionally, `matrix-python.yml` creates build matrix including multiple python versions. In which way are these different from `test_python_version.yml`?